src/ziggurat/messaging/consumer.clj

```diff
@@ -25,18 +25,23 @@
   [message topic-entity]
   (try
     (s/validate mp/message-payload-schema message)
-    (catch Exception e
+    (catch Exception _
       (log/info "old message format read, converting to message-payload: " message)
       (let [retry-count (or (:retry-count message) 0)
             message-payload (mp/->MessagePayload (dissoc message :retry-count) (keyword topic-entity))]
         (assoc message-payload :retry-count retry-count)))))
 
+(defn- try-deserialize-message
```

Comment:
we need to add at least a couple of unit tests to test this logic.

Reply:
This function gets tested indirectly by other unit tests